android/firestore-genrule-py-impl.py

```diff
@@ -0,0 +1,32 @@
+import json
```

Comment:
How about writing a FirestoreConfig.java file that reads the values from google-services.json?

Reply:
You can add a TODO for it and do it later.